lib/Factory/LayoutFactory.php

```diff
@@ -1037,6 +1028,31 @@ public function loadByJson($layoutJson, $playlistJson, $nestedPlaylistJson, Fold
                     }
                 }
 
+                $widgetProperties = [];
```

Comment:
Interesting way to handle this! Good thinking. I suggest we change the variable to `widgetConditionMatch` or something like that, so that it's clearer what we're doing.

This condition format is also controlled by us, so I think we can make our lives easier here and standardise on a `name==value` syntax (no quotes, etc).